src/backend/tests/users.test.ts

```diff
@@ -75,14 +76,25 @@ describe('Users', () => {
     expect(prisma.user.update).toHaveBeenCalledTimes(1);
   });
 
-  test('updateUserSettings', async () => {
-    const batmanSettings = {
-      id: 'bm',
-      userId: 1,
-      defaultTheme: Theme.DARK,
-      slackId: 'slack'
-    };
+  test('getUserSettings for undefined request user', async () => {
+    jest.spyOn(prisma.user, 'findUnique').mockResolvedValue(null);
+    const res = await request(app).get('/null/settings');
+    expect(res.statusCode).toBe(404);
+    expect(res.body.message).toBe(`user #NaN not found!`);
```

Comment:
hmmm id say pass in a real number because having the error include #NaN is weird and also we should probably have another error for if the param is null lol but that's another problem